test/integration/targets/setup_mongodb/tasks/main.yml

```diff
@@ -128,6 +129,19 @@
   when:
     - ansible_distribution == "Fedora"
 
+#- name: Fix corrupted yum db in Redhat 8 Shippable
+#  shell: rm -rf /var/lib/rpm/ && yum clean all
+#  when:
+#    - ansible_os_family == "RedHat"
+#    - ansible_distribution_version.split('.')[0]|int == 8
+#    - not ansible_distribution == "Fedora"
+
+- name: Ensure epel package is installed
+  package:
+    name: epel-release
+    state: present
+  when: ansible_os_family == "RedHat"
```

Comment:
If this test needs EPEL, it should use the `setup_epel` role.

Add `needs/target/setup_epel` to the `aliases` file for this task (this copies the role into the test execution environment). Then use `include_role` with the appropriate conditional.

Reply:
I want to say that this test should also remove the EPEL repo once it's done, but I don't see other tests doing that. It's probably best to add a handler to this test that removes the repo.